hack/ci/e2e-k8s.sh

```diff
@@ -137,14 +137,12 @@ run_tests() {
 
   # setting this env prevents ginkgo e2e from trying to run provider setup
   export KUBERNETES_CONFORMANCE_TEST='y'
-  # setting these is required to make RuntimeClass tests work ...
+  # setting these is required to make RuntimeClass tests work ... :/
```

Comment:
it is incredibly lame that we still need to do this. we should pull this from the node info or something.
... but that's another problem for another day.

Reply:
... these tests _also_ still require creating a magical `test-handler` runtime ahead of time in your CRI configuration (eg configuring containerd) so :|

something to keep in mind for the three of you working on conformance testing ...